### PR TITLE
Feature add new map script

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: scripts/readme.md [hunk 1]
```diff
@@ -14,15 +14,13 @@ To modify node positions:
   6. Copy final_map to map tiles and tile_config as the maps config for tile.js
 
 To add a map:
-  1. Do the same as above till step 3 for the new map
+  1. Copy exciting map as original map in maps folder
 
-  2. Add the map to the maps object and create a new json file in map
+  2. Copy exciting tile_config to tile_config file inside maps folder
```
Comment (rampr) [inline scripts/readme.md]: typo, It should be existing

File: scripts/readme.md [hunk 2]
```diff
@@ -14,15 +14,13 @@ To modify node positions:
   6. Copy final_map to map tiles and tile_config as the maps config for tile.js
 
 To add a map:
-  1. Do the same as above till step 3 for the new map
+  1. Copy exciting map as original map in maps folder
```
Comment (rampr) [inline scripts/readme.md]: typo, it should be existing

File: scripts/add_new_map.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+      return;
+    }
+
+    console.log("Starting with max ms: ", max_ms, "\nRepeat: ", repeat, "\nchapter: ", chapter);
+
+    old_config = JSON.parse(old_config.replace(/'/g,'"')
+      .replace(/(['"])?([a-z0-9A-Z_]+)(['"])?:/g, '"$2": '));
+
+    jsio.path.add('../');
+    jsio('import scripts.maps.original_map as original_map');
```
Comment (Javasamurai) [inline scripts/add_new_map.js]: Can we shift this statement to top

File: scripts/readme.md [hunk 4]
```diff
@@ -36,3 +34,8 @@ duplicate_map <1> <2> <3> (This scrip should be run from scripts folder)
   <2> name of the file to dump the computed map, if empty will dump
     at adventuremap/scripts/maps/final_map.js
   <3> location to map export config. default: adventuremap/scripts/map_config.json
+
+add_new_map <1> <2> <3> (This scrip should be run from scripts folder)
```
Comment (Javasamurai) [inline scripts/readme.md]: script

File: scripts/add_new_map.js (hunk 3 above)
Comment (vivekb91) [reply scripts/add_new_map.js]: we need to import jsio first after that only we can import. moving to top of the function.

**Review by rampr — APPROVED** [review]

**Pull request merged**